packages/perspective/src/js/perspective.node.js

```diff
@@ -30,46 +30,78 @@ delete module.exports["worker"];
 
 let CLIENT_ID_GEN = 0;
 
+const DEFAULT_ASSETS = [

```

Comment:
I think these URLs should be `require.resolve` so that the WebSocket host can be in a directory that is not the root of the package, e.g. I have a structure like:

```
/
   package.json
   src/
       server.js
```

for my server.

Reply:
Good call.

Reply:
@texodus Also might be an idea to try catch the plugins which are not part of `@jpmorganchase/perspective`.

Reply:
Turns out this was tricky to do, as none of these are deps of @jpmorganchase/perspective itself - ended up using resolve with a modified `paths` to point to `process.cwd()`.  This will work as long as all `@jpmorganchase/*`packages are hoisted in your project ... unclear of another way to do this unfortunately.